app/models/services/pubmed/resource.rb

```diff
@@ -25,8 +25,13 @@ def mapper
         title:              lambda {|data| data.css('ArticleTitle').text },
         publication:        lambda {|data| data.css('Journal Title').text },
         doi:                lambda {|data| data.css('ArticleId[IdType=doi]').text },
-        pubmed_id:          lambda {|data| data.css('ArticleIdList ArticleId[IdType=pubmed]').text },
-        abstract:           lambda {|data| data.css('AbstractText').map(&:text).join("\n\n") },
+        pubmed_id:          lambda {|data| data.css('PMID').text },
+        abstract:           lambda do |data|
+          data.css('AbstractText').map do |a|
+            [a['Label'],a.text]
```

Comment:
Looks like it just joins everything:

```ruby
[['whats','up'],['hey','there']].join ' '
#=> "whats up hey there"
```

That said, since this looks weird and is not obvious at first glance, we should probably change it.